src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -271,7 +268,7 @@ const QueryBuilderRow: React.FC<IQueryBuilderRowProps> = (props) => {
                     noOptionsText={t('advancedSearch.noOptionsFoundMessage')}
                     placeholderText={t('advancedSearch.valueFieldPlaceholder')}
                     resetInputOnBlur={false}
-                    searchPropertyName={DTID_PROPERTY_NAME}
+                    searchPropertyName={selectedProperty?.data.name || ''}
```

Comment:
is '' a valid search property name? should we keep DTID_PROPERTY_NAME in the right side of the ||?